<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateLipDetectionConditionWidget</class>
 <widget class="QWidget" name="CreateLipDetectionConditionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>300</height>
   </rect>
  </property>
  <widget class="QPushButton" name="pbSelect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>124</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start training</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbInverted">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>90</y>
     <width>139</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>An inverted condition is satisfied when its specified details are NOT true</string>
   </property>
   <property name="whatsThis">
    <string>An inverted condition is satisfied when its specified details are NOT true</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Invert Condition</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout"/>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>384</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Select the threshold value for lip detection:</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="lblThreshold">
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>Kindly adjust the threshold value to optimize whether
you are speaking or not by detecting lip movements. </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSlider" name="horizontalSlider">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-50</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lblSilderValue">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblNotification">
      <property name="text">
       <string>Notifications!</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pbOk">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>258</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblThreshold_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Kindly select the appropriate threshold value by
 training though your lip movements while speaking</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>splitter_3</zorder>
  <zorder>pbSelect</zorder>
  <zorder>cbInverted</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>lblThreshold_2</zorder>
 </widget>
 <tabstops>
  <tabstop>pbSelect</tabstop>
  <tabstop>cbInverted</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
